main/newscorm/learnpath.class.php

```diff
@@ -5497,7 +5502,7 @@ public function return_new_tree($update_audio = 'false', $drop_element_here = fa
         }
 
         $this->tree_array($arrLP);
```

Comment:
OK, aunque en este caso creo que se simplificaría todavía diciendo $this->arrMenu = null; en lugar de estas 2 líneas (pero habría que verificar como reacciona)